src/otx/core/model/entity/base.py

```diff
@@ -210,10 +213,20 @@ def export(
             base_name: (str): base name for the exported model file. Extension is defined by the target export format
             export_format (OTXExportFormatType): format of the output model
             precision (OTXExportPrecisionType): precision of the output model
+            dump_auxiliaries (bool): Flag to return "feature_vector" and "saliency_map". Defaults to False.
+
         Returns:
             Path: path to the exported model.
         """
-        return self._exporter.export(self.model, output_dir, base_name, export_format, precision)
+        exporter = self._exporter
+        if dump_auxiliaries:
+            self._reset_model_forward()
+            if hasattr(exporter, "onnx_export_configuration"):
```

Comment:
Now this check looks as a workaround, which arises because `_exporter` is configurable only from the object state.
In the future, we can introduce explain mode, which affects the object state and allows switching to explain forward & export parameters.
For instance, `OTXModel.set_explain_mode(enabled=True/False)`.
That'd allow to use that mode with `litModule.predict` as well to dump explain results into a dataset.